packages/tools/kadena-cli/src/tx/commands/txCreateTransaction.ts

```diff
@@ -66,40 +66,45 @@ export const createTransactionCommandNew = createCommandFlexible(
     globalOptions.outFileJson(),
   ],
   async (option, values) => {
-    const template = await option.template();
-    const templateVariables = await option.templateVariables({
-      values,
-      variables: template.templateConfig.variables,
-    });
-    const outputFile = await option.outFile({
-      values,
-      variables: template.templateConfig.variables,
-    });
-
-    if (IS_DEVELOPMENT) {
-      console.log('create-transaction:action', {
-        ...template,
-        ...templateVariables,
-        ...outputFile,
+    try {
+      const template = await option.template();
+      const templateVariables = await option.templateVariables({
+        values,
+        variables: template.templateConfig.variables,
+      });
+      const outputFile = await option.outFile({
+        values,
+        variables: template.templateConfig.variables,
       });
-    }
 
-    if (template.templateConfig.template === undefined) {
-      return console.log('template not found');
-    }
+      if (IS_DEVELOPMENT) {
+        console.log('create-transaction:action', {
+          ...template,
+          ...templateVariables,
+          ...outputFile,
+        });
+      }
+
+      if (template.templateConfig.template === undefined) {
+        return console.log('template not found');
+      }
 
-    const result = await createTransaction(
-      template.templateConfig.template,
-      templateVariables.templateVariables,
-      outputFile.outFile,
-    );
-    assertCommandError(result);
+      const result = await createTransaction(
+        template.templateConfig.template,
+        templateVariables.templateVariables,
+        outputFile.outFile,
+      );
+      assertCommandError(result);
 
-    console.log(result.data.transaction);
+      console.log(result.data.transaction);
 
-    const relativePath = path.relative(process.cwd(), result.data.filePath);
-    console.log(`\ntransaction saved to: ./${relativePath}`);
+      const relativePath = path.relative(process.cwd(), result.data.filePath);
+      console.log(`\ntransaction saved to: ./${relativePath}`);
 
-    return { outFile: relativePath };
+      return { outFile: relativePath };
+    } catch (error) {
```

Comment:
we should probably just do this try{}catch{} in the createCommand right? if its such a generic handling

Reply:
would be cleaner yeah,